.github/workflows/ci.yml

```diff
@@ -0,0 +1,151 @@
+        shell: bash -l {0}
+
+    env:
+      IS_MASTER_BUILD: ${{ !matrix.numfocus_nightly && matrix.os == 'ubuntu-latest' && matrix.pyarrow == '0.17.1' && matrix.python == '3.8' }}
+
+    steps:
+      # CI setup
+      - name: Is Master Build?
+        run: echo $IS_MASTER_BUILD
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Conda Bootstrap
+        uses: goanpeca/setup-miniconda@v1
+        with:
+          auto-update-conda: true
+          channel-priority: strict
+          channels: conda-forge
+          python-version: ${{ matrix.python }}
+      - name: Conda Config
+        run: |
+          conda config --set always_yes yes
+          conda config --set changeps1 no
+          conda config --set pip_interop_enabled True
+
+      # Installation into the `test` environment
+      - name: Conda Install Pyarrow (non-nightly)
+        run: conda install pyarrow==${{ matrix.pyarrow }}
+        if: matrix.pyarrow != 'nightly'
+      - name: Conda Install Pyarrow (nightly)
+        # Install both arrow-cpp and pyarrow to make sure that we have the
+        # latest nightly of both packages. It is sadly not guaranteed that the
+        # nightlies and the latest release would otherwise work together.
+        run: conda install -c arrow-nightlies arrow-cpp pyarrow
+        if: matrix.pyarrow == 'nightly'
+      - name: Conda Install conda-requirements.txt + conda-test-requirements.txt
+        # run this AFTER pyarrow, since this is less restrictive (so conda doesn't spend too much time downgrading)
+        run: conda install --file=conda-requirements.txt --file=conda-test-requirements.txt
+      - name: Pip Instal NumFOCUS nightly
+        # NumFOCUS nightly wheels, contains numpy and pandas
+        run: env PRE_WHEELS="https://pypi.anaconda.org/scipy-wheels-nightly/simple" pip install --pre --upgrade --timeout=60 -f $PRE_WHEELS pandas numpy
+        if: matrix.numfocus_nightly
+      - name: Pip Install blacken-docs
+        # currently not on conda
+        run: pip install --no-deps blacken-docs
+      - name: Pip Install kartothek
+        run: pip install --no-deps -e .
+      - name: Conda Export
+        run: conda list --export
+
+      # Linters
+      - name: Flake8
+        run: flake8
+      - name: Mypy
+        run: mypy .
+      - name: Black
+        run: black --check .
+      - name: blacken-docs
+        run: blacken-docs
+      - name: Isort
+        run: isort --recursive --check-only
+
+      # Tests
+      - name: Pytest
+        run: pytest --cov kartothek --cov-report xml
+      - name: ASV
+        run: |
+          asv --config ./asv_bench/asv.conf.json machine --machine travis --os unknown --arch unknown --cpu unknown --ram unknown
+          asv --config ./asv_bench/asv.conf.json run --show-stderr --environment existing --quick
+      - name: Sphinx
+        run: |
+          python setup.py build_sphinx
+          touch ./docs/_build/html/.nojekyll
```

Comment:
What's this?

Reply:
Copy+paste leftover from `rle-array`. **But stay with me:** there is it a marker to signal that the output is not jekyll-driven (and files with `_` are just ordinary files), so that the docs can easily by hosted on GitHub pages. Now the question is: do you want to switch from RTD to GitHub pages? This would also mean that we only need one single build pipeline and don't need to work with yet another build environment.

For reference: [the rle-array docs on GH pages](https://jdasoftwaregroup.github.io/rle-array/).

Reply:
@xhochy 🔝 any opinion on that?

Reply:
Well, I don't have a strong opinion there but if we change the hosting we'd need to change blog posts, etc. as well

Reply:
I can do that if we agree on that. It's just a quick search+replace.